typescript/packages/jumble/integration/utils.ts

```diff
@@ -39,3 +39,68 @@ export const login = async (page: Page) => {
   button = await page.$("button");
   await button!.click();
 };
+
+export const addCharm = async () => {
+  const space = `ci-${Date.now()}-${
+    Math.random().toString(36).substring(2, 15)
+  }`;
+  const process = new Deno.Command("deno", {
```

Comment:
Could use `Deno.execPath` in lieu of `"deno"` for shell-less path

Reply:
coooooool - I didn't realize this